src/lib/getKeyDownHandlers.ts

```diff
@@ -0,0 +1,50 @@
+import * as _ from 'lodash'
+import keyboardHandlerFilter from './keyboardHandlerFilter'
+import {
+  IAccessibilityDefinition,
+  AccessibilityActions,
+  ActionsHandler,
+} from 'src/lib/accessibility/interfaces'
+import { IState, IPropsWithVarsAndStyles } from '../../types/theme'
+
+/**
+ * Assigns onKeyDown handler to the Component's part element, based on Component's actions
+ * and keys mappings defined in Accessibility behavior
+ * @param {AccessibilityActions} actions Actions defined in a component.
+ * @param {IAccessibilityDefinition} accessibility The input element which is to loose focus.
```

Comment:
seems that description of this param is incorrect

Reply:
seems like I am blind :) thank you, people, for reading it!